fvm/evm/handler/coa.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
@zhangchiqing, @jordanschalm would love your feedback on this contract given your great level of knowledge in this area. it will be the contract deployed for cadence-owned accounts aka bridged accounts and It should work as if we have a basic smart contract wallet.

Reply:
It looks correct to me. Seems like the tricky part will be in `cadenceArch`. 

Some documentation on `cadenceArch`, `FlowReceived`, and `isValidSignature` would be beneficial.